packages/core/src/app/init.ts

```diff
@@ -16,6 +18,14 @@ import koaSpaProxy from '@/middleware/koa-spa-proxy';
 import initOidc from '@/oidc/init';
 import initRouter from '@/routes/init';
 
+const logListening = (port: number, https = false) => {
```

Comment:
nit: feels like pass in http/https protocol directly is fine? Could remove conditionalString